iree_tests/conftest.py

```diff
@@ -293,6 +295,9 @@ def __init__(self, spec, **kwargs):
         self.run_args.extend(self.spec.iree_run_module_flags)
         self.run_args.append(f"--flagfile={self.spec.data_flagfile_name}")
 
+        self.atol=1e-05
+        self.rtol=1e-06
```

Comment:
We can put comparison thresholds in the flagfiles themselves, rather than make that a property of the test runner